### PR TITLE
[unit_config] Replace alertRules.starage to alertRules.usedDisks and [aos_config] Replace resourceRatios with requestedResources

### PR DESCRIPTION
**Pull request by DmytroRozzuvailo**
1. [aos_config] Replacing resourceRatios with requestedResources - according to - [requirements](https://kb.epam.com/pages/viewpage.action?pageId=2002693996)

2. [unit_config] Replacing alertRules.storage with alertRules.usedDisks - according to - [Core's implementation](https://github.com/aosedge/aos_common/blob/feature_dynamic_nodes/aostypes/aostypes.go#L192) and created [issue](https://jiraeu.epam.com/browse/EPMPAOS-5420) with node config

### CODE REVIEW

File: src/cloud_common/protocols/unit/unit_config.py [hunk 1]
```diff
@@ -159,12 +172,12 @@ class AlertRules(BaseModel):
         ),
     ]
 
-    storage: Annotated[
-        AlertRulePercents,
+    used_disks: Annotated[
```
Comment (al1img) [inline src/cloud_common/protocols/unit/unit_config.py:175]: Let's name it just `disk` to match same in the monitoring struct.

Comment (DmytroRozzuvailo) [reply src/cloud_common/protocols/unit/unit_config.py]: updated. done

File: src/cloud_common/protocols/unit/unit_config.py [hunk 2]
```diff
@@ -172,11 +172,11 @@ class AlertRules(BaseModel):
         ),
     ]
 
-    used_disks: Annotated[
+    disks: Annotated[
         Optional[list[AlertRulePercentsOfDisk]],
         Field(
             default=None,
-            alias='usedDisks',
+            alias='disks',
```
Comment (al1img) [inline src/cloud_common/protocols/unit/unit_config.py:179]: `disk`

Comment (DmytroRozzuvailo) [reply src/cloud_common/protocols/unit/unit_config.py:179]: why 'disk' if it is a list?

Comment (DmytroRozzuvailo) [reply src/cloud_common/protocols/unit/unit_config.py]: done. disks -> disk

Comment (oleh-mykytiuk) [reply src/cloud_common/protocols/unit/unit_config.py]: Cloud can support both `disk` and `disks` for backward compatibility

File: src/cloud_common/protocols/unit/unit_config.py [hunk 3]
```diff
@@ -71,6 +71,19 @@ class AlertRulePercents(BaseModel):
     ]
 
 
+class AlertRulePercentsOfDisk(AlertRulePercents):
```
Comment (mykola-kobets-epam) [inline src/cloud_common/protocols/unit/unit_config.py:74]: suggested renaming AlertRulePartitionUseRatio
"""
Partition use ratio in percents
"""

Comment (DmytroRozzuvailo) [reply src/cloud_common/protocols/unit/unit_config.py:74]: I've merged these changes already into aos_cloud. So is it strong proposal?

Comment (mykola-kobets-epam) [reply src/cloud_common/protocols/unit/unit_config.py:74]: no. suggestion

**Pull request merged**